### PR TITLE
1 create note listing page UI

### PR DESCRIPTION
**Pull request by yashjais**

### CODE REVIEW

File: app/javascript/src/components/Common/Navbar/index.jsx [hunk 1]
```diff
@@ -1,54 +1,43 @@
...
+        label: "Contacts",
+        to: "/contacts",
+        icon: () => <UserCircle color="#1e1e20" size={24} />
+      },
+      {
+        label: "Settings",
+        to: "/my/profile",
+        icon: () => <Settings color="#1e1e20" size={24} />
+      }
+    ]}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/Navbar/index.jsx:26]: Please move this to a `contants.js` file.

File: app/javascript/src/components/Dashboard/Notes/ListNote.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    marginTop: "2px"
+  },
+  time: {
+    margin: "5.5px",
+    color: "#68737D"
+  },
+  avatar: {
+    margin: "7px"
+  }
+};
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/ListNote.jsx:113]: We don't need to do this. Please refer table styles in `https://github.com/bigbinary/neeto-ui/blob/master/lib/styles/components/_table.scss`.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 3]
```diff
@@ -1,97 +1,143 @@
 import React, { useState, useEffect } from "react";
 
+import { Search, Settings, Plus, BurgerMenu } from "@bigbinary/neeto-icons";
+import { Typography, Input, Button } from "@bigbinary/neetoui/v2";
+import { MenuBar, Header } from "@bigbinary/neetoui/v2/layouts";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:5]: Please use alias for `@bigbinary/neetoui/v2` as `neetoui`.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 4]
```diff
@@ -1,97 +1,143 @@
...
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui";
-import { Header, SubHeader } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
+import ListNote from "./ListNote";
+import { notes } from "../../../data/notes";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:9]: Can we keep this constant data inside a `constants.js` file inside the scope of `Notes` directory?

File: app/javascript/src/data/notes.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+export const notes = [
```
Comment (karthiknmenon) [inline app/javascript/src/data/notes.js:1]: `export const NOTES`

**Review by karthiknmenon — COMMENTED** [review]
@yashjais Please check the suggestions. Also, please refrain from adding branch numbers in PR titles.

File: app/javascript/src/components/Dashboard/Notes/index.jsx (hunk 3 above)
Comment (yashjais) [reply app/javascript/src/components/Dashboard/Notes/index.jsx:5]: Done.

**Pull request merged**